doc/how-to-generate.md

```diff
@@ -42,6 +66,18 @@ We have pre-prepared docker image for you to use autorest.az easily. (Please mak
     > azdev test {service_name}
     ```
 
+5. Run linter check on generated command
+    * run azdev linter
+    ``` bash
+    > azdev linter {service_name}
+    ```
+
+6. Run style check on generated code
+    * run azdev style
+    ``` bash
+    > azdev style {service_name}
+    ```
```

Comment:
please merge this with test to make things look simpler.thx

Reply:
resolve

---
In reply to: [449505426](https://github.com/Azure/autorest.az/pull/461#discussion_r449505426) [](ancestors = 449505426)